test/utils/shippable/other.sh

```diff
@@ -5,6 +5,12 @@ set -o pipefail
 retry.py apt-get update -qq
 retry.py apt-get install -qq \
     shellcheck \
+    g++-4.9 \
```

Comment:
Merge botch? Lines 8,9,12 and 13 were already removed in r. 0587d5926685a6ae004b8d5065ed1a803b13386f. Why are they back?